content/pt-br/about/privacy.md

```diff
@@ -10,7 +10,7 @@ Esta política de privacidade explicará como nossa organização utiliza os dad
 
 A InnerSource Commons coleta a maioria dos dados diretamente fornecidos por você. Coletamos e processamos dados quando você:
 
-- Se inscreve em nossa lista de e-mails: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.
+- Se inscreve em nossa newsletter: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.
```

Comment:
- Cadastre-se em nossa newsletter: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.